source/fab/metrics.py

```diff
@@ -160,7 +160,7 @@ def metrics_summary(metrics_folder: Path):
     step_names = ['preprocess fortran', 'preprocess c', 'compile fortran']
     for step_name in step_names:
         if step_name not in metrics:
-            logger.error(f'no step metrics called {step_name}')
+            # logger.error(f'no step metrics called {step_name}')
```

Comment:
If code isn't needed then remove it. Otherwise it is needed and shouldn't be commented. Is the problem that the error message needs to be reworded for clarity?